### PR TITLE
Irda: builds and installs

### PR DESCRIPTION
**Pull request by vmoravec**

### CODE REVIEW

File: patches/irda.patch [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ # Makefile.am for YCP module .../doc/autodocs
+-
+-include $(top_srcdir)/autodocs-ycp.ami
++# FIXME
++# TODO autodocs temporarily off
++# include $(top_srcdir)/autodocs-ycp.ami
+diff --git a/src/Makefile.am b/src/Makefile.am
+index c40d1e4..0e3ae86 100644
+--- a/src/Makefile.am
++++ b/src/Makefile.am
```
Comment (jreidinger) [inline patches/irda.patch]: this is wrong, you should not patch makefile as it will be regenerated by yk makefile command

Comment (vmoravec) [inline patches/irda.patch]: Without patching of this makefile the build fails. This is also a standard workflow for other modules' autodocs, have a look at other pull requests.

Comment (lslezak) [inline patches/irda.patch]: More specifically, all `Makefile.am` files under `src/` are removed by the YCP-killer, you do not need to update them, only the `Makefiles.am` **outside** `src/`.

Comment (vmoravec) [inline patches/irda.patch]: OK, fix pushed.

**Pull request merged**